### ISSUE TITLE
Version 1.7.0 can't be minified with `create-react-app`

### ISSUE THREAD

**Issue by matthieualouis** [rhysd/react-component-octicons#1]
I have build an app using `create-react-app` (using a TS variant of it. Specifically `react-scripts-ts` version 2.13.0), and when running the latest version of `react-component-octicons`, I am getting a build time error when `create-react-app` tries to minify the code: 

```
Failed to minify the code from this file:

 	./node_modules/react-component-octicons/index.js:4

Read more here: http://bit.ly/2tRViJ9

error Command failed with exit code 1.
```

This worked fine on version 1.6.0, let me know if you need any more information :)

**Comment by rhysd** [comment]
I compared 1.6.0 and 1.7.0. In 1.6.0, `index.js` was compiled to ES5, but in 1.7.0, `index.js` was compiled to ES2015 (or later). I did not change any compiler options. So, I guess, the default target ES version was changed by bumping up `typescript` version.
I guess create-react-app is using uglifyjs as minifier, and it only supports ES5. It's why this issue occurred.

**Comment by rhysd** [comment]
@matthieualouis I published new version 1.7.1. Could you try the latest?

**Comment by matthieualouis** [comment]
Hi @rhysd that worked! Thanks a lot for the quick turnaround :)
